### DISCUSSION TITLE
Auto-empty trophy and weight field upon selection in catchbook

### DISCUSSION THREAD

**Discussion by Errandir-of-Gondolin** [janstehno/cotw-companion#90] [Ideas]
Not sure if this is the case on all platforms, but on my Android version of the app, whenever I select the trophy or weight field in the catchbook, I first have to delete the automatically filled in 0 before I can enter the value. It would be more intuitive/save keystrokes if, when selecting one of these input fields, that initial 0 is automatically removed.

**Comment by janstehno** [comment]
Hey, thank you for the suggestion. This is because the app need some data to be saved for the trophy and weight so it defaults to 0. I could probably just remove the zeros entirely or I could remove them as soon as user taps the text field. Either way, as you wrote, it does not make sense. I will look into it.
